src/libs/actions/Transaction.ts

```diff
@@ -102,39 +94,39 @@ function saveWaypoint(transactionID, index, waypoint) {
         return;
     }
 
-    const recentWaypointAlreadyExists = _.find(recentWaypoints, (recentWaypoint) => recentWaypoint.address === waypoint.address);
+    const recentWaypointAlreadyExists = recentWaypoints.find((recentWaypoint) => recentWaypoint?.address === waypoint?.address);
     if (!recentWaypointAlreadyExists) {
-        const clonedWaypoints = _.clone(recentWaypoints);
+        const clonedWaypoints = lodashClone(recentWaypoints);
         clonedWaypoints.unshift(waypoint);
         Onyx.merge(ONYXKEYS.NVP_RECENT_WAYPOINTS, clonedWaypoints.slice(0, 5));
     }
 }
 
-function removeWaypoint(transactionID, currentIndex) {
+function removeWaypoint(transactionID: string, currentIndex: string) {
```

Comment:
Are you sure that `index` is a `string`?
```suggestion
function removeWaypoint(transactionID: string, currentIndex: number) {
```

Reply:
@rezkiy37 The below line is saying that index is string
```
// Index comes from the route params and is a string
```